prow/config/external_plugins_config.yaml

```diff
@@ -75,6 +75,7 @@ ti-community-label:
       - type
       - found
       - impact
+      - report
```

Comment:
```suggestion
      - report
      - impact
```

Reply:
DITO for others.

Reply:
Also you should update the labels.yaml and labels.md file(update this file by run `./scripts/update-labels.sh` command)